### ISSUE TITLE
DS does not launch

### ISSUE THREAD

**Issue by rth3** [rg35xx-cfw/rg35xx-cfw.github.io#26]
Attempting to open a DS ROM on v0.6 will result in a black screen then return to home.

**Comment by acmeplus** [comment]
Yes, currently desmume and melons are broken, however see #25 to see how to launch drastic instead.

**Comment by zopzop76** [comment]
I've tried every version of Batocera for the RG35xx and none of them launched a DS rom.  I made sure each installation was a clean flash and not just an update.  Every other game system works, even the PSP and Dreamcast but the NDS roms refuse to play.  I checked the ROMs on my Windows PC and they played just fine.  I have no idea how people are playing NDS roms on the RG35xx.

**Comment by ANhan88** [comment]
> #25 để biết cách khởi chạy mạnh mẽ thay thế.
